[2025-02-02T21:13:34.689+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T21:13:34.707+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flights_pipeline.transform_data manual__2025-02-02T21:13:24.301450+00:00 [queued]>
[2025-02-02T21:13:34.715+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flights_pipeline.transform_data manual__2025-02-02T21:13:24.301450+00:00 [queued]>
[2025-02-02T21:13:34.717+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 2
[2025-02-02T21:13:34.729+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): transform_data> on 2025-02-02 21:13:24.301450+00:00
[2025-02-02T21:13:34.735+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=5521) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T21:13:34.736+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'flights_pipeline', 'transform_data', 'manual__2025-02-02T21:13:24.301450+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/zdag.py', '--cfg-path', '/tmp/tmpd6xl0o9a']
[2025-02-02T21:13:34.737+0000] {standard_task_runner.py:72} INFO - Started process 5553 to run task
[2025-02-02T21:13:34.737+0000] {standard_task_runner.py:105} INFO - Job 100: Subtask transform_data
[2025-02-02T21:13:34.782+0000] {task_command.py:467} INFO - Running <TaskInstance: flights_pipeline.transform_data manual__2025-02-02T21:13:24.301450+00:00 [running]> on host d628bd12729b
[2025-02-02T21:13:34.851+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flights_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T21:13:24.301450+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-02T21:13:24.301450+00:00'
[2025-02-02T21:13:34.867+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T21:13:35.114+0000] {logging_mixin.py:190} INFO - Données transformées avec succès.
[2025-02-02T21:13:35.123+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T21:13:35.132+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T21:13:35.133+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=flights_pipeline, task_id=transform_data, run_id=manual__2025-02-02T21:13:24.301450+00:00, execution_date=20250202T211324, start_date=20250202T211334, end_date=20250202T211335
[2025-02-02T21:13:35.154+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-02-02T21:13:35.172+0000] {taskinstance.py:3891} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T21:13:35.176+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
